api/admin/problem_details.py

```diff
@@ -304,12 +304,19 @@
 )
 
 INVALID_EMAIL = pd(
-    "http://librarysimplified.org/terms/problem/invalid-configuration-option",
+    "http://librarysimplified.org/terms/problem/invalid-email",
     status_code=400,
     title=_("Invalid email format"),
     detail=_("You must enter a valid email address."),
 )
 
+INVALID_URL = pd(
+    "http://librarysimplified.org/terms/problem/invalid-url",
+    status_code=400,
+    title=_("Invalid URL format"),
+    detail=_("You must enter a valid URL."),
+)
+
```

Comment:
Maybe @aslagle has an opinion, but I'd be okay with all of these "bad format" problem details having the same URI.

Reply:
OK, let's use `invalid-format` for all of these going forward.